.github/workflows/arviz_compat.yml

```diff
@@ -2,16 +2,22 @@ name: arviz-compatibility
 
 on:
   pull_request:
+    paths:
+      - ".github/workflows/*.yml"
+      - "pymc/**.py"
+      - "*.py"
+      - "conda-envs/**"
+      - "codecov.yml"
+      - "scripts/*.sh"
   push:
     branches: [main]
     paths:
-      - ".github/workflows/*"
-      - "pymc/**"
-      - "setup.py"
-      - "pyproject.toml"
-      - "buildosx"
```

Comment:
The `pyproject.toml` is only relevant for the tools running in pre-commit.

The `buildosx` does not seem to have a purpose. It's 8 years old, was never changed and is not mentioned anywhere in the codebase.